### ISSUE TITLE
Make it multisite compatible

### ISSUE THREAD

**Issue by PSahner** [resmio/wp-reservation-plugin#1]
On a WordPress multisite network the button and widget itself work fine but you can not use the extended options (like colors and such).

**Comment by nsb** [comment]
Do you know what the issue is?

**Comment by PSahner** [comment]
No, have to take a closer look. I guess it has to do something with the stylesheet.
